lib/roll/app_builder.rb

```diff
@@ -79,11 +79,11 @@ def configure_generators
     end
 
     def configure_mailers_preview_path
-      config = <<-RUBY
+      config = <<-'RUBY'
 
 
   # Specific mailers path
-  config.action_mailer.preview_path = Rails.root + '/spec/mailers/previews'
+  config.action_mailer.preview_path = "#{Rails.root}/spec/mailers/previews"
```

Comment:
Could be `Rails.root.join('spec/mailers/previews')`.
We won't need to disable interpolation for heredoc syntax.